### PR TITLE
Create 2023-11-23.md

### PR DESCRIPTION
**Pull request by robcraig-LF**
Adds meeting minutes and agenda SPDX AI Team Meeting 2023-11-23

### CODE REVIEW

File: ai/SPDX AI Team Meeting 2023-11-23.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# SPDX AI Team Meeting 2023-11-23
+## Attendees
+*  Kate
+
+## Agenda
+* Welcome Jean's PhD (*Gurjar, Ambarish Aniruddha <agurjar@iu.edu) about sbom library with visualization presentation
+* Update on Fosdem CFP? - Kate
+*Round Table Discussion
```
Comment (kestewart) [inline ai/SPDX AI Team Meeting 2023-11-23.md:8]: space before bullet

File: ai/SPDX AI Team Meeting 2023-11-23.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
+# SPDX AI Team Meeting 2023-11-23
+## Attendees
+*  Kate
+
```
Comment (kestewart) [inline ai/SPDX AI Team Meeting 2023-11-23.md:4]: Hmm...  there was definitely more than just me at this one.

**Comment by kestewart** [comment]
@bennetkl - where did you put the updates?  on the etherpad?   If so @robcraig-LF - can you go pull in the updates to your branch and fix the formatting that's been highlighted?

File: ai/SPDX AI Team Meeting 2023-11-23.md (hunk 1 above)
Comment (bennetkl) [reply ai/SPDX AI Team Meeting 2023-11-23.md:8]: I updated with all attendees, but not in this pull request.

**Pull request merged**